src/main/java/picard/illumina/CollectIlluminaLaneMetrics.java

```diff
@@ -92,6 +98,9 @@
     @Option(shortName = "EXT", doc="Append the given file extension to all metric file names (ex. OUTPUT.illumina_lane_metrics.EXT). None if null", optional=true)
     public String FILE_EXTENSION = null;
 
+    @Option(doc = "Boolean the determines if this run is a NovaSeq run or not. (NovaSeq tile metrics files are in cycle 25 directory.", optional = true)
+    public boolean IS_NOVASEQ = false;
```

Comment:
typically we leave that up to the caller which may have more information that we have available. (the normal case is zamboni determining this based on machineType)